blueprints/roulette.py

```diff
@@ -26,11 +30,27 @@ def spin():
 
     if int(user_data['balance']) >= int(content['bet']):
         result = random.randint(0, 37)
-        if content['choice'] == "red" and result in red or content['choice'] == "black" and result in black or \
-                content['choice'] == "green" and result in green:
-            bet_value = int(content['bet'])
+        if content['type'] == "number":
+            if int(content['number']) is None:
+                return "invalid number"
+            elif int(content['number']) == result:
+                bet_value = 36 * int(content['bet'])
+            else:
+                bet_value = -int(content['bet'])
+        elif content['choice'] is None:
+            return "invalid choice"
+        elif content['type'] == "evenodd":
+            if content['choice'] == "even" and result in even or content['choice'] == "odd" and result in odd:
```

Comment:
Könntest auch im if prüfen ob result gerade oder ungerade ist. Dann sparst du dir das speichern in der Liste.